scripts/static-bootstrap.sh

```diff
@@ -0,0 +1,174 @@
+export S3_BUCKET
+export S3_ENDPOINT
+
+SPACE=$(echo $VCAP_APPLICATION | jq -r '.["space_name"]')
+WWW_HOST=${WWW_HOST:-$(echo $VCAP_APPLICATION | jq -r '.["application_uris"][]' | grep 'www\.usa\.gov' | tr '\n' ' ')}
+WWW_HOST=${WWW_HOST:-$(echo $VCAP_APPLICATION | jq -r '.["application_uris"][]' | grep -v 'apps.internal' | grep -E 'beta|dev-dr' | tr '\n' ' ')}
+CMS_HOST=${CMS_HOST:-$(echo $VCAP_APPLICATION | jq -r '.["application_uris"][]' | grep -E 'cms|shared-egress-dr' | tr '\n' ' ')}
+if [ -z "$WWW_HOST" ]; then
+  WWW_HOST="*.app.cloud.gov"
+fi
+if [ -z "$CMS_HOST" ]; then
+  CMS_HOST=$(echo $VCAP_APPLICATION | jq -r '.["application_uris"][]' | head -n 1)
+fi
+export WWW_HOST
+export CMS_HOST
+
+S3_ROOT_WEB=${S3_ROOT_WEB:-/web}
+S3_ROOT_CMS=${S3_ROOT_CMS:-/cms/public}
+S3_HOST=${S3_HOST:-$S3_BUCKET.$S3_ENDPOINT}
+S3_PROXY_WEB=${S3_PROXY_WEB:-$S3_HOST$S3_ROOT_WEB}
+S3_PROXY_CMS=${S3_PROXY_CMS:-$S3_HOST$S3_ROOT_CMS}
+S3_PROXY_PATH_CMS=${S3_PROXY_PATH_CMS:-/s3/files}
+export S3_ROOT_WEB
+export S3_ROOT_CMS
+export S3_HOST
+export S3_PROXY_WEB
+export S3_PROXY_CMS
+export S3_PROXY_PATH_CMS
+
+#if [ -f "/etc/php81/php-fpm.d/env.conf.tmpl" ]; then
+#  cp /etc/php81/php-fpm.d/env.conf.tmpl /etc/php81/php-fpm.d/env.conf
+#  echo "env[S3_PROXY_PATH_CMS] = "$S3_PROXY_PATH_CMS >> /etc/php81/php-fpm.d/env.conf
+#  echo "env[S3_PROXY_CMS] = "$S3_PROXY_CMS >> /etc/php81/php-fpm.d/env.conf
+#  echo "env[S3_ROOT_CMS] = "$S3_ROOT_CMS >> /etc/php81/php-fpm.d/env.conf
+#  echo "env[S3_HOST] = "$S3_HOST >> /etc/php81/php-fpm.d/env.conf
+#fi
+
+export DNS_SERVER=${DNS_SERVER:-$(grep -i '^nameserver' /etc/resolv.conf|head -n1|cut -d ' ' -f2)}
+
+export EN_404_PAGE=${EN_404_PAGE:-/page-error/index.html};
+export ES_404_PAGE=${ES_404_PAGE:-/es/pagina-error/index.html};
+
+export NEW_RELIC_DISPLAY_NAME=${NEW_RELIC_DISPLAY_NAME:-$(echo $SECRETS | jq -r '.NEW_RELIC_DISPLAY_NAME')}
+export NEW_RELIC_APP_NAME=${NEW_RELIC_APP_NAME:-$(echo $SECRETS | jq -r '.NEW_RELIC_APP_NAME')}
+export NEW_RELIC_API_KEY=${NEW_RELIC_API_KEY:-$(echo $SECRETS | jq -r '.NEW_RELIC_API_KEY')}
+export NEW_RELIC_LICENSE_KEY=${NEW_RELIC_LICENSE_KEY:-$(echo $SECRETS | jq -r '.NEW_RELIC_LICENSE_KEY')}
+
+
+SP_KEY=$(echo $SECAUTHSECRETS | jq -r '.spkey')
+SP_CRT=$(echo $SECAUTHSECRETS | jq -r '.spcrt')
+
+# seems not to be needed
+#spkey=$(echo "$SP_KEY" | awk '{gsub(/\\n/, "\n")}1')
+#spcrt=$(echo "$SP_CRT" | awk '{gsub(/\\n/, "\n")}1')
+
+echo "$SP_KEY" > /var/www/sp.key
+echo "$SP_CRT" > /var/www/sp.crt
+
+ENV_VARIABLES=$(awk 'BEGIN{for(v in ENVIRON) print "$"v}')
+# this overwrites the files in place, so be careful mounting in docker
+echo "Inserting environment variables into nginx config templates ... "
+for FILE in /etc/nginx/*/*.conf.tmpl /etc/nginx/*.conf.tmpl; do
+    if [ -f "$FILE" ]; then
+        OUTFILE=${FILE%.tmpl}
+        echo " generating $OUTFILE"
+        envsubst "$ENV_VARIABLES" < "$FILE" > "$OUTFILE"
+    fi
+done
+
+# update new relic with environment specific settings
```

Comment:
This whole newrelic-related block shouldn't be needed, but since it has an "if" clause around it I'll leave it be for now.